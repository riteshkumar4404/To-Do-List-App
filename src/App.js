import React from "react";
import Routes from "../src/translations/routes/index";
export default props => <>{Routes}</>;
