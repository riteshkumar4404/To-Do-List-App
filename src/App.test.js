import React from "react";
import App from "./App";
import { shallow } from "enzyme";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });
describe("App Compnent", () => {
  it("should render without error", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
