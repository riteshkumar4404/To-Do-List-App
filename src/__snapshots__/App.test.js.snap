// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Compnent should render without error 1`] = `ShallowWrapper {}`;
